python/smashbox/utilities/__init__.py

```diff
@@ -26,6 +27,32 @@ def setup_test():
     If exception is raised then the testcase execution is aborted and smashbox terminates with non-zero exit code,
 
     """
+    cmd = config.oc_sync_cmd
+    if cmd[0 - len(' --trust'):] == ' --trust':
+        cmd = cmd[:0 - len(' --trust')]
```

Comment:
Yes, i reported it to the client team, but for now I check for it and remove it. Basically it can only have one option atm